src/lib.rs

```diff
@@ -98,6 +99,7 @@ pub mod pallet {
 		/// Checker for the reward vec, is it initalized already?
 		type Initialized: Get<bool>;
 		/// Percentage to be payed at initialization
+		#[pallet::constant]
```

Comment:
What determines which lines get this macro?

Reply:
The one below I would say, I just checked with polkadot-JS which of the types in https://substrate.dev/rustdocs/v3.0.0/src/pallet_balances/lib.rs.html#18-1370 are shown

Reply:
(e.g., existentialDeposit)